fastflix/widgets/panels/status_panel.py

```diff
@@ -10,21 +11,25 @@
 from fastflix.language import t
 from fastflix.models.fastflix_app import FastFlixApp
 from fastflix.models.video import Video
-from fastflix.shared import time_to_number
+from fastflix.shared import time_to_number, timedelta_to_str
 
 logger = logging.getLogger("fastflix")
 
 
 class StatusPanel(QtWidgets.QWidget):
     speed = QtCore.Signal(str)
     bitrate = QtCore.Signal(str)
+    signal_elapsed_time = QtCore.Signal(str)
```

Comment:
`signal_elapsed_time = QtCore.Signal()`

If not transmitting a value can have an blank signal